icubam/messaging/sms_sender.py

```diff
@@ -75,7 +75,7 @@ def get(config, sms_carrier=None):
   that is if the sender object is 'XXSender' the carrier name should be 'XX'
   """
   sms_carrier = sms_carrier or config.sms.carrier
-  cls_name = "{}Sender".format(sms_carrier)
+  "{}Sender".format(sms_carrier)
```

Comment:
If we don't use it, maybe there is no reason to do this formatting. Or just `str(sms_carrier)` to check that `__repr__` works?

Reply:
I'll let others judge but it seems that the line can be wholly removed indeed?